LDAR_Sim/src/utils/generic_functions.py

```diff
@@ -207,46 +207,46 @@ def ecef_to_llh(ecef_km):
     return lat, lon, h
 
 
-def init_orbit_poly(predictor, T1, T2, interval):
```

Comment:
Why are you removing the satellite functionality? If it is not called, it won't be used, right?